tests/test_aws_aurora_support.py

```diff
@@ -0,0 +1,57 @@
+from unittest.mock import Mock
+
+from pytest import mark, raises
+from sqlalchemy import text
+
+
+@mark.sqlalchemy("1.4")
```

Comment:
`postgresql_readonly` is only available in SQLAlchemy >=1.4. [Ref](https://docs.sqlalchemy.org/en/14/dialects/postgresql.html#setting-read-only-deferrable)